src/webapp/project_reader.py

```diff
@@ -20,27 +26,36 @@ def get_projects(indir="../projects"):
             continue
         project = OrderedDict.fromkeys(["ID", "Name", "Description", "PIName", "Organization", "Department",
                                         "FieldOfScience", "Sponsor"])
-        project.update(load_yaml_file(os.path.join(indir, file)))
+        try:
+            data = load_yaml_file(os.path.join(indir, file))
+        except yaml.YAMLError:
+            if strict:
+                raise
+            else:
+                log.error("skipping")
```

Comment:
Could we make this more specific to describe what we're skipping and why?

Reply:
Same goes for all the other "skipping" messages

Reply:
The reason there's nothing more specific is because load_yaml_file() already logs the specifics and I didn't want the duplication.

Reply:
Well we could say that we're skipping it instead of failing because it's running in non-strict mode